docs/cypress.md

```diff
@@ -166,3 +167,14 @@ already familiar with Cypress.
 
 This is a small selection - the Cypress best practices guide, linked above, has more good advice, and we
 should generally try to adhere to them.
+
+## Percy Visual Testing
+We also support visual testing via Percy, this extracts the DOM from Cypress and renders it using custom renderers
+for Safari, Firefox, Chrome & Edge, allowing us to spot visual regressions before they become release regressions.
+Right now we run it as part of the standard Pull Request CI automation but due to only having 25k screenshots/month,
+and each `cy.percySnapshot()` call results in 8 screenshots (4 browsers, 2 sizes) this could quickly be exhausted and
+at that point we would likely run it on a CRON interval or before releases.
```

Comment:
maybe we should wrap calls to percy in a flag that can be disabled so that we can separate out runs with percy? unless there's already a way?

Reply:
Percy will automatically no-op where Cypress is not ran from within percy, i.e `percy exec --` - so for local testing cypress will behave the same as without percy, except with some extra log lines

![image](https://user-images.githubusercontent.com/2403652/168611877-4ceabd22-d16b-4e52-b0f9-ec2b683cd0ec.png)

In future we may additionally also create a label e.g `X-Needs-Percy` to control whether a PR needs to run the percy suite